bundles/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/values/OnOffValue.java

```diff
@@ -44,55 +43,40 @@ public class OnOffValue extends Value {
      * Creates a switch On/Off type, that accepts "ON" for on and "OFF" for off.
      */
     public OnOffValue() {
-        this(OnOffType.ON.name(), OnOffType.OFF.name());
+        this(null, null);
     }
 
     /**
      * Creates a new SWITCH On/Off value.
      *
-     * values send in messages will be the same as those expected in incoming messages
-     *
-     * @param onValue The ON value string. This will be compared to MQTT messages. Defaults to "ON".
-     * @param offValue The OFF value string. This will be compared to MQTT messages. Defaults to "OFF".
+     * @param onValues The list of ON value strings. These will be compared to MQTT messages. Defaults to "ON" if null.
+     * @param offValues The list of OFF value strings. These will be compared to MQTT messages. Defaults to "OFF" if
+     *            null.
      */
-    public OnOffValue(@Nullable String onValue, @Nullable String offValue) {
```

Comment:
maintaining this constructor as a convenience seems quite useful for uses _besides_ the mqtt.generic binding (i.e. from within homie and homeassistant, and tests in general)